app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_branch_mixin.py

```diff
@@ -264,35 +264,73 @@ def test_no_duplicate_branch_names(self):
         )
 
     def test_no_treatment_branches_for_rollout(self):
-        experiment = NimbusExperimentFactory.create(
-            status=NimbusExperiment.Status.DRAFT,
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            application=NimbusExperiment.Application.DESKTOP,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
+            feature_configs=[
+                NimbusFeatureConfigFactory(
+                    application=NimbusExperiment.Application.DESKTOP
+                )
+            ],
+            is_sticky=True,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.MAC_ONLY,
         )
         experiment.is_rollout = True
         experiment.save()
         for branch in experiment.treatment_branches:
             branch.delete()
 
         data = {
+            "application": f"{NimbusExperiment.Application.DESKTOP}",
```

Comment:
Oh this is another one of *Jared's Patented Nits*.  If what you want is to convert something to a string, then `str(...)` is more idiomatic than wrapping it in a format string where you're not actually doing any formatting.

Reply:
Alternatively, if what you want is the string value of the enum, you can say `NimbusExperiment.Application.DESKTOP.value`